assignment3/Dictionary.java

```diff
@@ -0,0 +1,35 @@
+package com.katermar.assignment3;
+
+import java.util.HashSet;
+
+/**
+ * Created by katermar on 6/8/2017.
+ */
+public class Dictionary {

```

Comment:
No need to submit Dictionary class, but feel free to leave it here.

And since it's here, I will add some comments :)